src/accelerate/test_utils/scripts/test_script.py

```diff
@@ -526,21 +526,28 @@ def main():
     if state.local_process_index == 0:
         print("**Initialization**")
     init_state_check()
-    if state.local_process_index == 0:
```

Comment:
Or maybe change this to `process_index == 0` so it's only run on machine 0?

Reply:
That cleans it up some, but there's still the issue of `state.num_processes` will be 16 on two nodes of 8